test_scripts/RC/AUDIO_LIGHT_HMI_SETTINGS/commonRCmodules.lua

```diff
@@ -0,0 +1,439 @@
+---------------------------------------------------------------------------------------------------
+-- RC common module for AUDIO, LIGHT, HMI_SETTINGS modules
+---------------------------------------------------------------------------------------------------
+config.application1.registerAppInterfaceParams.appHMIType = { "REMOTE_CONTROL" }
+config.application2.registerAppInterfaceParams.appHMIType = { "REMOTE_CONTROL" }
+
+--[[ Required Shared libraries ]]
+local commonRC = require("test_scripts/RC/commonRC")
+local test = require("user_modules/dummy_connecttest")
+
+--[[ Local Variables ]]
+local c = {}
+c.modules = { "RADIO", "CLIMATE", "AUDIO", "LIGHT", "HMI_SETTINGS" }
+
+function c.preconditions()
+  commonRC.preconditions()
+end
+
+function c.start()
+  commonRC.start(nil, test)
+end
+
+function c.AddOnRCStatusToPT(tbl)
+  tbl.policy_table.functional_groupings.RemoteControl.rpcs.OnRCStatus = {
+    hmi_levels = { "NONE", "BACKGROUND", "FULL", "LIMITED" }
+  }
+end
+
+function c.getRCAppConfig()
+  local struct = commonRC.getRCAppConfig()
+  struct.moduleType = c.modules
+  struct.moduleType = { "RADIO", "CLIMATE" }
```

Comment:
@GetmanetsIrina ```struct.moduleType``` defined 2 times: in line 31 and in line 32

Reply:
updated in e69bc5b00e261ce2992688ec9d8d637f9377f372 commit.